README.md

```diff
@@ -170,6 +170,12 @@ e.g `make image-build USE_LOCAL=true"`
 
   The default image used is `quay.io/opendatahub/opendatahub-operator:dev-0.0.1` when not supply argument for `make image`
 
+- To build multi-arch image, by config env variablel PLATFORM
+  ```commandline
+  export PLATFORM=linux/amd64,linux/arm64,linux/ppc64le,linux/s390x
+  make image
```

Comment:
Is it really a requirement to get it from environment? I would rather keep it make-only (`make PLATFORM=...`) than depend of non-specific, but generic name in the environment.

Reply:
i set env in a local.mk file, so i do not need to pass PLATFORM everytime i type make.
and if i do not need multiple arch, i just comment out that variable in local.mk
i think we were using local.mk as mentioned in the https://github.com/opendatahub-io/opendatahub-operator/blob/main/docs/troubleshooting.md#using-a-localmk-file-to-override-makefile-variables-for-your-development-environment

Reply:
Using `PLATFORM ?=` make it possible to set the variable for the environment (in unix sense) so can be useful for people who use something like https://direnv.net. But in general it's a common place, so non-namespaced variables there ring the bell.

I'm just wondering what is the intention. Environment works for me as well.